pkg/agent/proxy/proxier.go

```diff
@@ -179,35 +210,40 @@ func (p *proxier) installServices() {
 	for svcPortName, svcPort := range p.serviceMap {
```

Comment:
This is not about your current change, but I am worried about the complexity of this func. Have we done any profiling to understand the CPU/memory costs of AntreaProxy with a large scale Services and Endpoints? @tnqn 

Have we considered any incremental computation approach, rather than checking every Service and endpoint for every update?

Of course, I do not mean any design change to do in this PR.

Reply:
Thanks for mentioning it.
As we discussed in https://github.com/vmware-tanzu/antrea/pull/1815/files#r572521304, I think it is OK to have a `O(n)` loop at least every second.

Reply:
Yes, I am still concerned in the large scale case, e.g. with 10K's of Services, what will be the CPBU/memory consumption of checking every Service, every 1s (assuming there are bursts of updates coming for a period). Even worse, it is happening on all Nodes.